### ISSUE TITLE
Billing History: add option to download receipts in PDF

### ISSUE THREAD

**Issue by Gustavo-Hilario** [Automattic/wp-calypso#56503]
### What

Add a "Download PDF" link in the billing history page.

### Why

When users are on the billing page they see two options: **view** or **email** their receipts. If they choose to view the receipt, they will be able to print it as a PDF, but for some users, it's hard to follow this path and some page headers may not be easily removed. If they prefer to email the receipt, they can print the email as a PDF too, but again, some headers and unnecessary content can still be shown.

The easiest route could be by adding a "Download PDF" link so users could quickly download their receipts and upload them to their accounting system.

### How

This should be something like this: **[screenshot](https://d.pr/i/06Hak1)**.

**Comment by Nic-Sevic** [comment]
also requested here it looks like: https://github.com/Automattic/wp-calypso/issues/52063

**Comment by cat-og** [comment]
User requested here as well: 9438248-zd-a8c

They noted as well that they receipt they see in their account says 'not an official VAT receipt', so only the emailed receipt was valid for their accountant. They would like the option to download the exact same receipt and information that was initially sent as a email, not the receipt visible on their account purchases.
